primitives/src/types.rs

```diff
@@ -71,6 +71,18 @@ pub type Block = generic::Block<Header, OpaqueExtrinsic>;
 /// Block ID.
 pub type BlockId = generic::BlockId<Block>;
 
+/// Auto-incrementing Uint
+/// Uniquely identifies a collection
+pub type CollectionId = u32;
+
+/// Auto-incrementing Uint
+/// Uniquely identifies a series of tokens within a collection
+pub type SeriesId = u32;
+
+/// Auto-incrementing Uint
+/// Uniquely identifies a token within a series
+pub type SerialNumber = u32;
```

Comment:
could you import these into nft/types and make sure they're used there

Reply:
nvm you did it already :)

Reply:
I think they already are imported into nft/types